edx/frontend-app-gradebook
# Split SearchControls into separate component

**TL;DR -** As part of the push to break up the Gradebook monolith, split the "search controls" into a separate component.

JIRA: [EDUCATOR-5663](https://openedx.atlassian.net/browse/EDUCATOR-5663)

**What changed?**

- Search controls (edit filters button and search box, highlighted in screenshot) moved to a new component
- Added test config and unit/snapshot tests for new components 😄 
- Updated package-lock format to new version

<img width="654" alt="Screen Shot 2021-04-22 at 11 47 29" src="https://user-images.githubusercontent.com/12944465/115744625-bbed2180-a360-11eb-9b9d-051ce3c96266.png">

**Developer Checklist**

**Testing Instructions**

See Gradebook test plan. Specifically, verify that the following workflows still work as intended:
- [x] Grades table results can be filtered from the "Filter" panel.
- [x] Users can be searched/filtered using the Search box.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:



FYI: @edx/masters-devs-gta
